src/graphql/query/loanCardDataExtended.graphql

```diff
@@ -0,0 +1,47 @@
+#import '../fragments/loanCardFieldsExtended.graphql'
+
+query loanCardData(
```

Comment:
This query seems to be the same as the one already in the project, `loanCardData`